src/test/scala/com/databricks/spark/avro/AvroSuite.scala

```diff
@@ -163,14 +172,14 @@ class AvroSuite extends FunSuite {
       }
     }
 
-    FileUtils.deleteDirectory(tempDir)
+    DirectoryDeletion.recursiveDelete(tempDir)
   }
 
   test("converting some specific sparkSQL types to avro") {
     val testSchema = StructType(Seq(StructField("Name", StringType, false),
-                                    StructField("Length", IntegerType, false),
+                                    StructField("Length", IntegerType, true),
```

Comment:
Nullable items are already a part of the previous test ("conversion to avro and back"), but I want this test to have nullable fields as well.